src/Chapter15/Listing15.20.AnotherInnerJoinWithSystem.Linq.Enumerable.Join.cs

```diff
@@ -24,12 +27,14 @@ public static void Main()
                     );
 
 
-            foreach ((long Id, string Name, Employee Employee) item in items)
+            foreach 
+                ((long Id, string Name, Employee Employee) item in items)
```

Comment:
I think this might be able to be one line?
```suggestion
            foreach ((long Id, string Name, Employee Employee) item in items)
```

Reply:
Again, this was too long to fit on a single line

Reply:
@MarkMichaelis thoughts?